.github/workflows/docs-ci-v2.yml

```diff
@@ -7,21 +7,36 @@ on:
     branches: [ 'main', 'release/v*' ]
 
 jobs:
+  makedirs:
+    if: ${{ github.event_name == 'push' }}
```

Comment:
This causes all the downstream jobs to get skipped it seems? I'm not sure if there is another form of dependency that says also runs if the parent job was not run?

Reply:
I think what I just checked in could work.  The "needs [makedirs]" really is for ordering.  So I moved the conditional to the steps.  So _makedirs_ should run regardless, but if it's not a push, the steps won't run.  I'm assuming the job is still seen as run even if no steps run?